hexacopter_params:
  ros__parameters:
    mass: 1.5  # kg
    inertia: [0.02, 0.02, 0.04, 0.00, 0.00, 0.00]  # kg·m² (Ixx, Iyy, Izz, Ixy, Ixz, Iyz)
    arm_length: 0.25  # meters
    k_thrust: 8.54858e-6  # N/(rad/s)^2
    k_torque: 1.6e-7  # Nm/(rad/s)^2
    max_rotor_speed: 800.0  # rad/s
    min_rotor_speed: 0.0  # rad/s
